OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/DataController.java

```diff
@@ -112,6 +112,12 @@ void addSensorToExperiment(final String experimentId, String sensorId,
     void removeSensorFromExperiment(final String experimentId, final String sensorId,
             MaybeConsumer<Success> onSuccess);
 
+    /**
+     * Completely removes a sensor reference from an experiment.
+     */
+    void eraseSensorFromExperiment(final String experimentId, final String sensorId,
```

Comment:
There is another implementation of this interface that need to be updated:
OpenScienceJournal/whistlepunk_library/src/sharedTest/java/com/google/android/apps/forscience/whistlepunk/StubDataController.java

Reply:
Fixed